### PR TITLE
LP-82: Install localgov_publications module for review

### PR DESCRIPTION
**Pull request by nbrettell**
## Include a summary of what this merge request involves (*)
Installs the localgov_publications module https://github.com/localgovdrupal/localgov_publications

## Call out any relevant implementation decisions
- Are there any links to back up your chosen methodology?
- Why have you taken the approach?
- Any particular problem areas?
## This PR has been tested in the following browsers
- [ ] Arc
- [ ] Edge
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] IE 11 (Windows)
- [ ] iOS Chrome
- [ ] iOS Safari
- [ ] Android Chrome
- [ ] Android Firefox
- [ ] Android default

### CODE REVIEW

File: config/default/oidc_auto_login.settings.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
-enabled: true
-client: nomensa
+_core:
+  default_config_hash: 0e6ehRzUiSqmn3UDZTPGqihtSWxeIvhwXy-VLg6i-fo
+enabled: false
+client: client
```
Comment (siliconmeadow) [inline config/default/oidc_auto_login.settings.yml:4]: You sure about this?

Comment (nbrettell) [reply config/default/oidc_auto_login.settings.yml:4]: No, thats just what `cex` gave me. Thought I had checked all other config files but looks like I missed this one, will remove

Comment (siliconmeadow) [reply config/default/oidc_auto_login.settings.yml:4]: Possibly a candidate for config_split, but then again...

Certainly not a time to fiddle with config split at the mo.

Comment (nbrettell) [reply config/default/oidc_auto_login.settings.yml]: removed

**Comment by Polynya** [comment]
Closing because the module is being tested on a essex.gov Review App

**Pull request closed without merging**